README.md

```diff
@@ -4,7 +4,7 @@ This repo holds the generated documentation for historical versions of NumPy.
 
 To add documentation for a version:
 
-- Run [`make dist`](http://www.numpy.org/devdocs/docs/howto_build_docs.html) in
+- Run [`make dist`](https://stackoverflow.com/questions/43197756/what-is-the-difference-between-make-and-make-dist) in
```

Comment:
Actually, here's the correct link:
```suggestion
- Run [`make dist`](https://numpy.org/devdocs/dev/howto_build_docs.html) in
```
but it doesn't necessarily explain make dist either (although I'm not sure if we have to).